### DISCUSSION TITLE
Add a delay to the Grid filters

### DISCUSSION THREAD

**Discussion by jmaster92** [vikramlearning/blazorbootstrap#150] [Ideas]
I have noticed that BlazorBootstrap really is very complete, but in the DataGrid component I cannot find the option to add a delay when the user types in the filters, since when a letter is typed the search is carried out immediately. A delay (a few miliseconds) would allow you to type a whole word or at least a set of characters before launching the search.

In this way, in the search on the server side, there would be much less requests.

Regards!

**Comment by gvreddy04** [comment]
@jmaster92 Will consider this in the next release.

**Reply by gvreddy04** [reply]
@jmaster92 New version 1.4.4 released!!!

Link: https://github.com/vikramlearning/blazorbootstrap/releases/tag/v1.4.4

**Reply by jmaster92** [reply]
I tested and now working with them, thank you. I'm your fan, man!
